Sprint 04

Heamex opened the pull request:
Всё поправил. Теперь приложение соответствует макету и не падает

Heamex committed:
Спритнт 4 закончен

Выполнена всё по спринту 4. Теперь приложение:
[main.storyboard]
- Имеет адаптивный, свёрстанный интерфейс.
Имеет две кнопки, два текстовых поля и картинку на главном экране.
[MovieQuizViewController]
- Показывает вопрос, номер вопроса и картинку
- После ответа на вопрос показывает рамку, цвет рамки зависит от правильности вопроса
- По окончании 10ти вопросов показывает простой результат со статистикой.

Heamex committed:
Внесены правки согласно фидбеку

Всё пофиксил, теперь приложение соответствует макету.

Heamex committed:
Мелкие правки

Добавлены комментарии для механизма временного выключения кнопок, а также добавлена настройка в MovieQuizViewController делающая статус-бар белым, как в дизайн-макете.

IBDesignable commented on MovieQuiz/Presentation/Base.lproj/Main.storyboard:
```diff
@@ -15,13 +24,124 @@
+                        </subviews>
                         <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
-                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <color key="backgroundColor" name="YP Background (iOS)"/>
```
цвет фона по макету YP Black

IBDesignable commented on MovieQuiz/Presentation/Base.lproj/Main.storyboard:
```diff
@@ -15,13 +24,124 @@
+                            <constraint firstItem="jMg-cj-MA1" firstAttribute="leading" secondItem="6Tk-OE-BBY" secondAttribute="leading" constant="20" id="L4H-bF-tho"/>
+                            <constraint firstItem="jMg-cj-MA1" firstAttribute="top" secondItem="6Tk-OE-BBY" secondAttribute="top" constant="10" id="PMy-rJ-RCZ"/>
+                            <constraint firstItem="6Tk-OE-BBY" firstAttribute="trailing" secondItem="jMg-cj-MA1" secondAttribute="trailing" constant="20" id="SuG-qF-4OE"/>
+                            <constraint firstItem="6Tk-OE-BBY" firstAttribute="bottom" secondItem="jMg-cj-MA1" secondAttribute="bottom" constant="20" id="qPI-yc-xNz"/>
```
Нижний констрейнт у стека должен быть 0

Если зажимать cmd при инспектировании отсутоп, увидешь что кнопки прижаты к нижней линии - safearea

Heamex committed:
Внесены правки согласно проверке от наставника

- Поправлен режим заполнения картинкой лаунч скрина.
- Заменён фон main на соответствующий макету.
- Исправлен нижний отступ VStack 20 -> 0
